README.md

```diff
@@ -21,15 +21,18 @@ and bound-constrained optimization
 This package provides an implementation of four classic algorithms for unconstrained/bound-constrained nonlinear optimization:
 
 - `lbfgs`: an implementation of a limited-memory BFGS line-search method for unconstrained minimization;
-    > Boggs, P. T., & Byrd, R. H. (2019). Adaptive, Limited-Memory BFGS
-    > Algorithms for Unconstrained Optimization. SIAM Journal on
-    > Optimization, 29(2), 1282-1299.
-    > DOI: [10.1137/16M1065100](https://doi.org/10.1137/16M1065100)
+    > D. C. Liu, J. Nocedal. (1989). On the limited memory BFGS method for
+    > large scale optimization. *Mathematical Programming*, 45(1), 503-528.
+    > DOI: [10.1007/BF01589116](https://doi.org/10.1007/BF01589116)
```

Comment:
It seems like the indentation of the reference here is not the same as in the other references.